src/ResourceManager/RecoveryServices/Commands.RecoveryServices/Vault/NewAzureRMRecoveryServicesVault.cs

```diff
@@ -56,13 +56,13 @@ public override void ExecuteCmdlet()
         {
             try
             {
-                VaultCreateArgs vaultCreateArgs = new VaultCreateArgs();
```

Comment:
Need to support ShouldProcess in this cmdlet, and need to specify the OutputType using an attribute